### ISSUE TITLE
Calendar card: Swap Weekly / Daily view buttons

### ISSUE THREAD

**Issue by bverkron** [home-assistant/frontend#7039]
## The request
The Weekly and Daily view buttons on the calendar card are reversed compared to the original one in the side panel. The typical Month > Week > Day order would make more sense IMO than the current Month > Day > Week order on the calendar card

## The alternatives
None

**Comment by zsarnett** [comment]
The calendar card has a different View than Week View. Its call List View. So its Month > Day > List

**Comment by bverkron** [comment]
Right, noticed that after submitting the issue. Devils advocate it's still scoped to a weekly timeframe to arguably it's still a weekly view.

Counter suggestion would be to add a 4th button for the original style weekly view and place it between the current monthly and daily buttons. Thoughts?
